### ISSUE TITLE
Simple test failing with "Unrecognized directive"

### ISSUE THREAD

**Issue by a-agmon** [VirtusLab/scala-cli#2079]
Although its clear Im doing something wrong, this is just so simple that I'm not sure:
I have a very simple scala test file - taken directly from the examples

filename - main.test.scala

```scala
//> using dep "org.scalatest::scalatest:3.2.10"
class SomeTest extends AnyFunSuite:
  test("Sometest.first") {
    val myNum = 10
    assert(myNum === 10)
}

```
When I try to run this using `scala-cli test main.test.scala` 
I get the following error 
```
(base) ➜  scala-cli-1 scala-cli test main.test.scala
[error] ./main.test.scala:2:16: Unrecognized directive: dep with values: org.scalatest::scalatest:3.2.10
[error] //> using dep "org.scalatest::scalatest:3.2.10"
[error]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Would be happy for guidance 

Scala CLI version: 0.1.17
Scala version (default): 3.2.0

**Comment by tgodzik** [comment]
Thanks for reporting! `dep` has been added in the newer versions of Scala CLI, I think it should be available in the latest stable `[0.2.1](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.1)` but also in the 1.0.0 RCs. Before that you could use `lib` instead.

**Comment by a-agmon** [comment]
Thanks, @tgodzik 
That works
